src/applications/personalization/profile360/components/MilitaryInformation.jsx

```diff
@@ -20,21 +23,57 @@ class MilitaryInformationContent extends React.Component {
       }
     } = this.props.militaryInformation;
 
-    if (error || serviceHistory.length === 0) return <LoadFail information="military"/>;
+    if (error) {
+      if (some(error.errors, ['code', '403'])) {
+        return (
+          <AlertBox
+            isVisible
+            status="warning"
+            content={<div>
+              <h3>We can’t access your military information</h3>
+              <p>We’re sorry. We can’t find your Department of Defense (DoD) ID. We need this to access your military service records. Please call us at 1-800-827-1000, or visit your nearest VA regional benefit office and request to be added to the Defense Enrollment Eligibility Reporting System (DEERS).</p>
+              <a href="/facilities">Find your nearest VA regional benefit office</a>.
+              <p>You can also request to be added to DEERS through our online customer help center.</p>
+              <a href="https://iris.custhelp.va.gov/app/answers/detail/a_id/3036/~/not-registered-in-deers%2C-or-received-and-error-message-while-trying-to">Get instructions from our help center</a>.
+            </div>}/>
+        );
+      } else if (some(error.errors, (e) => {
+        return ['500', '503'].includes(e.code);
+      })) {
+        return <LoadFail information="military"/>;
+      }
+    }
 
-    return (
-      <div data-field-name="serviceHistory">
-        {serviceHistory.map((service, index) => {
-          return (
-            <div key={index}>
-              <h3>{service.branchOfService}</h3>
-              <div>{moment(service.beginDate).format('MMM D, YYYY')} &ndash; {moment(service.endDate).format('MMM D, YYYY')}</div>
-            </div>
-          );
-        })}
-      </div>
-    );
+    if (this.props.veteranStatus.servedInMilitary) {
+      if (serviceHistory.length === 0) {
```

Comment:
Is it necessary to check `servedInMilitary` here? I see we're checking it in the higher-level component already, so it seems like it's guaranteed to be true by the time we reach this point.